### PR TITLE
feat(nix-shell): merging both nix-shells into one in root folder

### PR DESCRIPTION
**Pull request by grdddj**
Addressing issue #48 - combining both nix-shells into one in the root folder, to make it less confusing

### CODE REVIEW

**Comment by grdddj** [comment]
Fixup commit c6e0bf4:
- reflecting the change in README

File: shell.nix [hunk 1]
```diff
@@ -1,13 +1,31 @@
...
-    wget
-  ];
-}
+# the last successful build of nixos-20.09 (stable) as of 2020-10-11
+with import
+  (builtins.fetchTarball {
+    url = "https://github.com/NixOS/nixpkgs/archive/0b8799ecaaf0dc6b4c11583a3c96ca5b40fcfdfb.tar.gz";
+    sha256 = "11m4aig6cv0zi3gbq2xn9by29cfvnsxgzf9qsvz67qr0yq29ryyz";
+  })
+{ };
```
Comment (tsusanka) [inline shell.nix:9]: @mmilata if we wish to update to latest build of nixos-21.05 (stable) we'll modify the commit to the latest one in `release-21.05` [branch](https://github.com/NixOS/nixpkgs/tree/release-21.05)? Or is it a different one?

Comment (mmilata) [reply shell.nix:9]: Preferably in `nixos-21.05`, `release-21.05` is the same but before passing CI. I replace the sha256 with 52 zeroes, then replace it again with the one `nix-shell` prints in the error message. Or you can use `nix-prefetch-url` to get the hash.

Comment (tsusanka) [reply shell.nix:9]: Good trick :). So I got commit `e9148dc1c30e02aae80cc52f68ceb37b772066f3` and hash `1xs5all93r3fg4ld13s9lmzri0bgq25d9dydb08caqa7pc10f5ki`. So let's try that?

Comment (grdddj) [reply shell.nix:9]: Yes, seems to work fine, I am about to commit these changes. Thanks for the help

**Comment by grdddj** [comment]
Commit 7edfdd5:
- adding latest version of Nixos build into the shell.nix, version 21.05

**Review by tsusanka — APPROVED** [review]
Great. Let's squash the fixup commit and let's merge :).

**Review by tsusanka — APPROVED** [review]

**Comment by tsusanka** [comment]
So the job seems to pass! https://gitlab.com/satoshilabs/trezor/trezor-suite/-/jobs/1399269501

So we have successfully added a commit into the `incoming` branch that does not seem to break anything! Or I messed up the "testing" but hopefully not :).

**Comment by grdddj** [comment]
> So the job seems to pass! https://gitlab.com/satoshilabs/trezor/trezor-suite/-/jobs/1399269501
> 
> So we have successfully added a commit into the `incoming` branch that does not seem to break anything! Or I messed up the "testing" but hopefully not :).

Good job! Nice that we have this testing pipeline to be more confident.

**Pull request merged**